packages/share-transfer/src/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { promiseTimeout } from "@toruslabs/http-helpers";
+
+export async function getClientIp(): Promise<string> {
```

Comment:
@chaitanyapotti Is there a better way to do this ?

Reply:
Maybe we can write API on torus-backed to return with some custom parsing. We have getIP function there.

Reply:
use the get from http-helpers

Reply:
This code won't work cause the method is GET and fetch defaults to POST

Reply:
no, default method i GET for fetch api.  https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

also here i cannot use http-helpers because this api return response in text/plain format but http-helpers get method implementation works with json response only.

Reply:
Ah ok cool